core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -903,9 +905,13 @@ class WorkspaceService(protected val ctx: RawlsRequestContext,
         .collect { case Success(workspaceId) =>
           workspaceId
         }
-        .toSeq
+      v2WorkspaceIdsForUser <- dataSource.inTransaction { dataAccess =>
```

Comment:
Modifying `workspaceQuery.getTags` to limit to v2 workspaces was unattractive because the tag database does not store the workspace version. I didn't really want to do a join there, so I'm first filtering the user's workspaces to v2 workspaces.